### ISSUE TITLE
in NSW DS, one cannot operate the main nav with the keyboard

### ISSUE THREAD

**Issue by eschweitzer78** [eschweitzer78/gps-design-systems-lwc#115]
### Summary

One cannot operate the main nav with the keyboard, leading to sub-par accessibility.

### Salesforce Org Type

Sandbox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by eschweitzer78** [comment]
Actually possible with tab.
